new_frontend/src/pages/early-pickup/index.tsx

```
@@ -38,59 +51,52 @@ function StudentProfileWithFilters({ student }: { student: Student }) {
       image={student.image}
-      name={student.name}
       student_name={student.student_name}
-      classSection={student.classSection}
       reference_number={student.reference_number}
       custom_division={student.custom_division}
       first_name={student.first_name}
       last_name={student.last_name}
       program_name={student.program_name}
       isLoading={classLoading}
-      children={<EarlyPickupChild studentId={student.name} program={student.program} />}
+      children={<EarlyPickupChild studentId={student.name} program={student.program} setLoading={setLoading} />}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)